Description: Fix Library Path
 Fix the path for ice_cube in spec_helper.rb
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2014-12-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -5,7 +5,7 @@
   # okay
 end
 
-require File.dirname(__FILE__) + '/../lib/ice_cube'
+require 'ice_cube'
 
 DAY = Time.utc(2010, 3, 1)
 WEDNESDAY = Time.utc(2010, 6, 23, 5, 0, 0)
